src/index.md

```diff
@@ -15,7 +15,7 @@ home: false
 The ecosystem consist of various components:
  - [QGIS](https://qgis.org/) > Powerful GIS Desktop application 
  - [QGIS Mergin Plugin](https://plugins.qgis.org/plugins/Mergin/) > <QGISPluginName />
- - [Mergin Cloud](https://public.cloudmergin.com) > SaaS Cloud Service (available also as Mergin CE)
+ - [Mergin Cloud](https://merginmaps.com) > SaaS Cloud Service (available also as Mergin CE)
  - [Mergin Maps Input](https://inputapp.io) > the <MobileAppName /> for iOS and Android
```

Comment:
https://inputapp.io is also fully redirected to https://merginmaps.com now, so best to get rid of it to everywhere